Source/Scene/OrthographicFrustum.js

```diff
@@ -312,6 +317,51 @@ define([
     };
 
     /**
+     * Returns the pixel's width and height in meters.
+     *
+     * @param {Number} drawingBufferWidth The width of the drawing buffer.
+     * @param {Number} drawingBufferHeight The height of the drawing buffer.
+     * @param {Cartesian2} result The object onto which to store the result.
+     * @param {Number} [distance=near plane distance] The distance to the near plane in meters.
+     * @returns {Cartesian2} The modified result parameter or a new instance of {@link Cartesian2} with the pixel's width and height in the x and y properties, respectively.
+     *
+     * @exception {DeveloperError} drawingBufferWidth must be greater than zero.
+     * @exception {DeveloperError} drawingBufferHeight must be greater than zero.
+     *
+     * @example
+     * // Example 1
+     * // Get the width and height of a pixel.
+     * var pixelSize = camera.frustum.getPixelDimensions(canvas.clientWidth, canvas.clientHeight, new Cartesian2());
+     */
+    OrthographicFrustum.prototype.getPixelDimensions = function(drawingBufferWidth, drawingBufferHeight, result, distance) {
```

Comment:
`distance` is used in `PerspectiveOffCenterFrustum` so we need to keep it.  Put it before the `result` parameter since the result parameter is always last.